src/App.test.tsx

```diff
@@ -92,7 +92,6 @@ describe('App component', () => {
       });
 
       const result = screen.queryByText(/Result/);
-
```

Comment:
This change seems unrelated

Reply:
Do you want a line break there? Elsewhere in the file it is also done like that. Declaration and then the expect line.

Reply:
I was wondering because this change is in an unrelated file bloating the diff. But we can keep it.